CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 Magento Functional Testing Framework Changelog
 ================================================
+3.0.0
+-----
+### Upgrade Instructions
+* Run `bin/mftf reset --hard` to remove old generated configurations.
```

Comment:
The next step should be `bin/mftf build:project`, right?

Reply:
Are you saying instead of `upgrade`?

Reply:
Before I include `upgrade` in build, I always run build first. What do you think?

Reply:
I see, you kind of *have* to after resetting.